File: Practice/src/introduction_coding_tests/AdditionOfFractions.py
def solution(numer1, denom1, numer2, denom2):
    answer = [numer1 * denom2 + numer2 * denom1, denom1 * denom2]
    num = 2
    while num <= answer[0] and num <= answer[1]:
        if answer[0] % num == 0 and answer[1] % num == 0:
            answer[0] /= num
            answer[1] /= num
        else:
            num += 1
    return answer

File: Practice/src/introduction_coding_tests/AdditionOfFractions.js
function solution(numer1, denom1, numer2, denom2) {
    var answer = [numer1 * denom2 + numer2 * denom1, denom1 * denom2];
    for (let i = 2; i <= answer[0] && i <= answer[1]; ) {
        if (answer[0] % i == 0 && answer[1] % i == 0) {
            answer[0] /= i;
            answer[1] /= i;
        } else {
            i++;
        }
    }
    return answer;
}
